scst-store/sca_scans/sca-scanning-results.md

```diff
@@ -1,4 +1,53 @@
 # SCA scanning results
+## <a id='110'></a>1.1.1
+
+### <a id='black-duck-ba'></a>Black Duck Binary Analysis (BDBA)
+
+#### <a id='api-backend'></a>API backend
+
+* Date: March 22, 2022
+* Results: No known vulnerabilities
+
+#### <a id='cli-sr'></a>CLI
+
+* Date: March 22, 2022
+* Results: No known vulnerabilities
+
+### <a id='grype-sr'></a>Grype
+
+Version: 0.33.1
+
+#### <a id='api-backend-ci'></a>API Backend Container Image
+
+* Date: March 22, 2022
+* Results: No critical, and 1 high vulnerabilities in 2 different packages [CVE-2022-0778](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-0778). Multiple medium and low vulnerabilities. For more information, see the [API Backend Container Image v1.1.1 CycloneDX file content](api-backend-container-image-v1.1.1-grype-result.md).
+
+#### <a id='api-backend-cr'></a>API Backend Code Repository
+
+* Date: March 22, 2022
+* Results: No known vulnerabilities
+
+#### <a id='cli-cr'></a>CLI Repository
+
+* Date: March 22, 2022
+* Results: No critical, and 2 high vulnerabilities: [CVE-2015-5237](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-5237), [CVE-2021-22570](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-22570). No medium or low vulnerabilities. Multiple medium and low vulnerabilities. For more information, see the [CLI Repository v1.1.1 CycloneDX file content](cli-repository-v1.1.1-grype-result.md).
```

Comment:
I think you forgot to remove the `No medium or low vulnerabilities.` phrase here. The next phrase says that there are some.